reference/docs-conceptual/PowerShell-Core-Support.md

```diff
@@ -0,0 +1,68 @@
+# PowerShell Core Support Lifecycle
+
+PowerShell Core is a distinct set of tools and components that ships separately from Windows PowerShell.
+Therefore, PowerShell Core is not included in the Windows 10 or Windows Server licensing agreements.
```

Comment:
Should we replace `Windows 10` with `Windows Desktop` or `Windows 7/8/10`?

Reply:
yeah, good feedback.